libs/sync/src/test/java/com/rudderstack/android/sync/utils/TestLogger.kt

```diff
@@ -33,6 +33,6 @@ class TestLogger : Logger {
     override fun error(tag: String, log: String, throwable: Throwable?) {
     }
 
-    override val level: Logger.LogLevel
-        get() = Logger.LogLevel.Error
-}
\ No newline at end of file
+    override val level: RudderLogger.LogLevel
+        get() = RudderLogger.LogLevel.ERROR
```

Comment:
🤔 Regarding the `TestLogger` should we probably have the initial value the most descriptive one, which is debug?